python/KAPEL.py

```diff
@@ -81,6 +87,29 @@ def summary_message(config, year, month, wall_time, cpu_time, n_jobs, first_end,
     )
     return output
 
+def individual_message(config, pod_name, memory, cores, wall_time, cpu_time, start_time, end_time):
+    """ Write an APEL individual job message based on prometheus metrics from a single pod """
+    output = (
+        f'APEL-individual-job-message: v0.3\n'
+        f'Site: {config.site_name}\n'
+        f'VO: {config.vo_name}\n'
+        f'SubmitHost: {config.submit_host}\n'
+        f'MachineName: {pod_name}\n'
+        f'LocalJobId: {pod_name}\n'
+        f'InfrastructureType: {config.infrastructure_type}\n'
+        f'ServiceLevelType: si2k\n'
+        f'ServiceLevel: {config.benchmark_value * 250}\n'
```

Comment:
This would oblige Kuantifier users in GRACC mode to put in some dummy benchmark value, although GRACC won't read it. I think it would be better to omit than to require a value that won't be used. I would be okay with simply commenting out ServiceLevelType and ServiceLevel here since the individual messages are not used in APEL mode, or maybe something like
```
benchmark_output = those two lines of f strings if config.benchmark_value is defined, else empty string
```
and then include `benchmark_output` in the printing of `output`. What do you think @mwestphall  ?

Reply:
I would agree, but think that splitting out the GRACC and APEL specific config values might make sense to address in a separate PR. `KAPELConfig.py` will also currently throw an error if `BENCHMARK_VALUE` is non-null.

Reply:
Okay sure, a separate PR sounds good.